cockatrice/src/player.cpp

```diff
@@ -905,13 +900,13 @@ void Player::actOpenDeckInDeckEditor()
 
 void Player::actViewGraveyard()
 {
-    static_cast<GameScene *>(scene())->toggleZoneView(this, "grave", -1);
```

Comment:
Why?

Reply:
CLion told me scene() could be a problem so we should dynamically cast it instead.